client/src/components/ThankYou/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import * as ROUTES from "../../constants/routes";
+import { Link } from "react-router-dom";
+
+class ThankYou extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <div>
+        <div>
+          <h1>Thank You </h1>
+          <h3>Would you like to go to?</h3>
+        </div>
+        <Link to={ROUTES.SETTINGS}>
```

Comment:
Can you either add a little space between the two buttons or stack them on top of each other?

Reply:
Done